otio_cookelensmetadata/adapter.py

```diff
@@ -130,9 +144,16 @@ def _read_from_documents(
             # RecordType are disregarded for LMF processing.
             continue
 
+        # If the omit_kinematic flag is set to True, skip any record type
+        # containing a kinematic type.
+        if omit_kinematic:
+            if record_type.is_kinematic:
```

Comment:
nitpick - I'd combine this into one line:

`if omit_kinematic and record_type.is_kinematic:`

This keeps the code to the left as much as possible and avoids people reading it pushing multiple depths of mental context.

Reply:
Oh, of course.  Good call.